Data/Data_vs170.vcxproj

```diff
@@ -340,6 +340,7 @@
       <TreatWChar_tAsBuiltInType>true</TreatWChar_tAsBuiltInType>
       <ForceConformanceInForLoopScope>true</ForceConformanceInForLoopScope>
       <RuntimeTypeInfo>true</RuntimeTypeInfo>
+      <AdditionalOptions>$(AdditionalCompilerOptions)</AdditionalOptions>
```

Comment:
@chrisbednarski all of these `AdditionalOptions` for vs160 and 170 are redundant;
https://github.com/pocoproject/poco/blob/4a9285c997234bc216657ec8f1ce1b23b66fd223/Data/Data_vs170.vcxproj#L348

vs140 and 150 will be [removed](https://github.com/pocoproject/poco/issues/4274) for 1.13.0 release